From d80d585282e5d6bbe6065001b6b944b96fbbdbc9 Mon Sep 17 00:00:00 2001
From: Markus Pargmann <mpa@pengutronix.de>
Date: Sun, 7 Jul 2013 15:13:29 +0200
Subject: [PATCH 053/161] ARM: dts: imx27 phyCARD-S SOM remove wrong i2c
 sensor

This sensor was introduced in the original pca100 board file, but
phyCARD-S SOM and RDK do not have a temperature sensor.

Signed-off-by: Markus Pargmann <mpa@pengutronix.de>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts b/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
index 01ee807..cdff8c7 100644
--- a/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
+++ b/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
@@ -50,11 +50,6 @@
 &i2c2 {
 	status = "okay";
 
-	lm75@4a {
-		compatible = "national,lm75";
-		reg = <0x4a>;
-	};
-
 	at24@52 {
 		compatible = "at,24c32";
 		pagesize = <32>;
-- 
1.7.10.4

